src/nerf/gun.gleam

```diff
@@ -11,11 +11,19 @@ pub external type StreamReference
 
 pub external type ConnectionPid
 
-pub fn open(host: String, port: Int) -> Result(ConnectionPid, Dynamic) {
-  open_erl(charlist.from_string(host), port)
+pub fn open(
+  host: String,
+  port: Int,
+  opts_map: Dynamic,
+) -> Result(ConnectionPid, Dynamic) {
+  open_erl(charlist.from_string(host), port, opts_map)
 }
 
-pub external fn open_erl(Charlist, Int) -> Result(ConnectionPid, Dynamic) =
+pub external fn open_erl(
```

Comment:
Oh dear, this function should be private. That was a mistake of mine. Would you mind also making this private?